ext/dom/tests/DOMXPath_quote.phpt

```diff
@@ -0,0 +1,84 @@
+--TEST--
+Test DOMXPath::quote with various inputs
+--SKIPIF--
+<?php if (!class_exists('DOMXPath')) die('skip DOMXPath not available.'); ?>
+--FILE--
+<?php
+$dom = new DOMDocument();
+$xpath = new DOMXPath($dom);
+
+
+/**
+ * Quote a string for use in an XPath expression.
+ * 
+ * Example: $xp->query("//span[contains(text()," . $xp->quote($string) . ")]")
+ * 
+ * @param string $string string to quote.
+ * @return string quoted string.
+ */
+function UserlandDOMXPathQuote(string $string): string
```

Comment:
I'm not sure this is actually necessary as the expected output has to be added by hand (and checked by hand) manually too. But I don't object.

Reply:
OK for me.